pkg/aggregate/aggregate.go

```diff
@@ -58,14 +59,56 @@ var aggrToString = map[AggrType]string{
 	aggrTypeStddev: "stddev", aggrTypeStdvar: "stdvar", aggrTypeAll: "*",
 }
 
+var aggrToSchemaField = map[string]config.SchemaField{
+	"count":  {Name: "count", Type: "array", Nullable: true, Items: "double"},
+	"sum":    {Name: "sum", Type: "array", Nullable: true, Items: "double"},
+	"sqr":    {Name: "sqr", Type: "array", Nullable: true, Items: "double"},
+	"max":    {Name: "max", Type: "array", Nullable: true, Items: "double"},
+	"min":    {Name: "min", Type: "array", Nullable: true, Items: "double"},
+	"last":   {Name: "last", Type: "array", Nullable: true, Items: "double"},
+	"avg":    {Name: "avg", Type: "array", Nullable: true, Items: "double"},
+	"rate":   {Name: "rate", Type: "array", Nullable: true, Items: "double"},
+	"stddev": {Name: "stddev", Type: "array", Nullable: true, Items: "double"},
+	"stdvar": {Name: "stdvar", Type: "array", Nullable: true, Items: "double"},
+}
+
+func SchemaFieldFromString(aggregators []string, col string) ([]config.SchemaField, error) {
+	fieldList := make([]config.SchemaField, 0, len(aggregators))
+	for _, s := range aggregators {
+		if s != "" {
+			trimmed := strings.TrimSpace(s)
+			if trimmed == "*" {
+				fieldList = make([]config.SchemaField, 0, len(aggrToSchemaField))
+				for _, val := range aggrToSchemaField {
+					fieldList = append(fieldList, getAggrFullName(val, col))
+				}
+				return fieldList, nil
+			} else {
+				field, ok := aggrToSchemaField[trimmed]
+				if !ok {
+					return nil, fmt.Errorf("invalid aggragator type %s", trimmed)
```

Comment:
please quote the '%s' so in case of "sum, ,avg" it will be clear what went wrong or/and just skip empty strings